whois-api/src/test/java/net/ripe/db/whois/api/ElasticSearchHelper.java

```diff
@@ -0,0 +1,67 @@
+package net.ripe.db.whois.api;
+
+import net.ripe.db.whois.common.dao.jdbc.DatabaseHelper;
+import org.apache.http.HttpHost;
+import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.indices.CreateIndexRequest;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ElasticSearchHelper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseHelper.class);
+
+    public static void setupElasticIndexes(final String indexName, final String metaDetaIndex) throws Exception {
+
+        try(final RestHighLevelClient esClient = getEsClient()) {
+            if(!isElasticRunning(esClient)) {
+                return;
+            }
+
+            CreateIndexRequest whoisRequest = new CreateIndexRequest(indexName);
+            esClient.indices().create(whoisRequest, RequestOptions.DEFAULT);
+
+            CreateIndexRequest whoisMetaDataRequest = new CreateIndexRequest(metaDetaIndex);
+            esClient.indices().create(whoisMetaDataRequest, RequestOptions.DEFAULT);
+        }
+    }
+
+    public static void resetElasticIndexes(final String indexName, final String metaDetaIndex) throws Exception {
+        try(final RestHighLevelClient esClient = getEsClient()) {
+
+            if(!isElasticRunning(esClient)) {
+                return;
+            }
+
+            try {
+                DeleteIndexRequest whoisRequest = new DeleteIndexRequest(indexName);
+                esClient.indices().delete(whoisRequest, RequestOptions.DEFAULT);
+            } catch (Exception ignored) {
+            }
+
+            try {
+                DeleteIndexRequest metadataRequest = new DeleteIndexRequest(metaDetaIndex);
+                esClient.indices().delete(metadataRequest, RequestOptions.DEFAULT);
+            } catch (Exception ignored) {
+            }
+        }
+    }
+
+    @NotNull
+    private static RestHighLevelClient getEsClient() {
+        return new RestHighLevelClient(RestClient.builder(new HttpHost(System.getProperty("elastic.host"), Integer.parseInt(System.getProperty("elastic.port")))));
```

Comment:
Turn System.getProperty() calls into static fields at the top of the class so it's clearer (or make the class a Spring component and inject the values).